### PR TITLE
files organised

### PR DESCRIPTION
**Pull request by aarpit1010**
__3__ 

#### Short description of what this resolves:

files are organised according to extensions.

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-__ISSUENUMBER__
```
Comment (deepakDOTexe) [inline .github/pull_request_template.md]: Please do not make changes to this file. you only need to submit the script

**Comment by aarpit1010** [comment]
fixed

File: Submissions/aarpit1010/FileOrg.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
```
Comment (deepakDOTexe) [inline Submissions/aarpit1010/FileOrg.py]: Make Task_2 folder and then put the script inside it

**Comment by aarpit1010** [comment]
added the script in Task_2 folder

**Comment by deepakDOTexe** [comment]
@orichamaru Please check this PR

**Comment by orichamaru** [comment]
@aarpit1010 use relative paths

**Comment by deepakDOTexe** [comment]
@aarpit1010 You have one more day to have this PR merged. After that I will close it due to no activity.

**Pull request closed without merging**